e2e/testcafe-devextreme/tests/scheduler/workSpace.ts

```diff
@@ -360,3 +360,40 @@ test('[T716993]: should has horizontal scrollbar with multiple resources and fix
 }).after(async () => {
   await removeStylesheetRulesFromPage();
 });
+
+test('Scheduler appointments should change color on update resources', async (t) => {
+  await t.wait(2000);
```

Comment:
We should avoid waiting. TestCafe has embedded functionality that can automatically wait expected conditions.